NEWS.md

```diff
@@ -5,6 +5,11 @@
 * Do not use `getOption("available_packages_filters")` option when calling
   `available.packages()`. (#1002)
 
+* Improve reporting of errors returned by shinyapps.io. (#997)
```

Comment:
Merge problem?

Reply:
No. I didn't see NEWS for your error change, so I added it here...

Reply:
Oh got it.